.vib/clickhouse/goss/clickhouse.yaml

```diff
@@ -0,0 +1,29 @@
+command:
+  check-clickhouse-server:
+    exec: timeout --preserve-status 5 clickhouse-server 2>&1 || true
+    exit-status: 0
```

Comment:
After our last discussion, we agreed to avoid using `|| true` when possible and prefer to explicitly check the status. Could you please update it here?

Reply:
In this case, the `|| true` is necessary as the command output varies from `0` and `143` seemingly at random. Increasing the timeout doesn't solve this and thus I decided tu force the `exit-status=0`. I'll a comment on this test.